src/instance/minecraft/chat-manager.ts

```diff
@@ -123,4 +129,35 @@ export default class ChatManager extends EventHandler<MinecraftInstance> {
       message
     })
   }
+
+  proceedFiltering(author: string, content: string): string {
+    let filteredMessage: string
+    try {
+      filteredMessage = this.clientInstance.app.profanityFilter.clean(content)
+    } catch {
+      /*
+        profanity package has bug.
+        will throw error if given one special character.
+        example: clean("?")
+        message is clear if thrown
+      */
+      filteredMessage = content
+    }
```

Comment:
You copied the code from discord chat manager. It is a duplicate code. Just create a function and put it in `shared-util.ts` instead of copying the code along with its comment.